_learning_hours/legacy/domain_observability.md

```diff
@@ -0,0 +1,108 @@
+---
+theme: refactoring
+title: Domain Oriented Obervability on Reservation Kata
+name: domain_oriented_observability
+kata: reservation
+difficulty: 2
+author: pierrickblons
+---
+
+# Domain Oriented Obervability
+
+It's useful to have logging and metrics in our code to understand how it's behaving in production. But sometimes those little calls to loggers or metrics class start to spread all across the code base. It's quite common to see methods with more code dedicated to observability than code actually solving the problem.
+
+In this learning hour we will see how to improve the understandability of code that requires observability.
+
+### Learning Goals
+
+- Improving the readability of the code
+- Refactoring techniques to extract code into methods
+
+### Session Outline
+
+* 5 min connect: Discussion around logging and metrics
+* 15 min concept: Introduce domain oriented observability refactoring
+* 30 min concrete: Do Reservation Kata in pairs
+* 10 min conclusions: Discussion about this technique in your current code base
+
+### Connect: What is observability?
+Based on the [Pick only the correct items](https://sammancoaching.org/activities/connect/pick_the_correct_items_on_the_list.html) activity 
+
+Share to the group the some propositions to answer the question "What is observability?".
+
+Example:
```

Comment:
Could you give the facilitator a clue what the expected correct answers might be?